appendix/backup-and-restore/docker-compose.rst

```diff
@@ -0,0 +1,43 @@
+Backup & Restore (Docker)
+=========================
+
+.. toctree::
+   :hidden:
+
+   docker-file-handling
```

Comment:
Should we move this to the end of the page, after the Backup and Restore sections?

Reply:
If it's too complex, then we can just bring it back to the main document.

But hold on, being a subsection of restore seems to be correct, no?